src/Serilog/Formatting/Display/MessageTemplateTextFormatter.cs

```diff
@@ -40,7 +40,7 @@ public class MessageTemplateTextFormatter : ITextFormatter
     public MessageTemplateTextFormatter(string outputTemplate, IFormatProvider? formatProvider = null)
     {
         Guard.AgainstNull(outputTemplate);
-
+          
```

Comment:
remove null diff (is there not an `.editorconfig` rule about this?

Reply:
Done